content/08-cardano-sidechains/03-example-evm-sidechain/04-sidechain-token.mdx

```diff
@@ -0,0 +1,21 @@
+---
+title: SC_Token and faucet
+metaTitle: SC_Token and faucet
+---
+
+## What is SC_Token?
+
+**SC_Token<sup>1</sup>** is the native token of the Cardano EVM sidechain Testnet. SC_Token can be moved between the main chain (Cardano) and the example EVM sidechain, and SC_Token is required to pay for gas.
+
+**Test SC_Token**, or 'Sidechain Testnet Token' SC_Token, is the native token of the example EVM sidechain testnet. Test SC_Token carries no real-world value.
+
+## How to get SC_Token?
+
+IO Global maintains a faucet at <PENDING_URL> for testing purposes. The faucet dispenses test SC_Token that you can use to pay the gas fees on smart contracts that you run on the EVM sidechain testnet. You can claim one test SC_Token every 24 hours. You cannot swap the test SC_Token tokens you claim from the faucet for any other token, nor can you use them to make any purchases.  
```

Comment:
IO Global maintains a faucet at https://faucet.cardano.evmtestnet.iohkdev.io/basic-faucet for testing purposes...